packages/@adobe/spectrum-css-temp/components/table/skin.css

```diff
@@ -288,10 +298,18 @@ tbody.spectrum-Table-body {
     background-color: var(--spectrum-table-divider-border-color);
   }
 
+  /* don't want the divider to add to the resizer's width since it's */
   &:active,
   &:focus-ring {
     &::after {
-      background-color: var(--spectrum-global-color-blue-400);
+      background-color: unset;
```

Comment:
I've got an element that sits on top and isn't the button, when focused, I don't want it visible, it causes a tiny wiggle because the anti-aliasing is strong than it should be, so this looks better